src/oc_erchef/apps/oc_chef_wm/test/oc_chef_action_tests.erl

```diff
@@ -301,7 +301,8 @@ end_to_end_test_() ->
     State = #base_state{requestor = #chef_requestor{name = <<"rob">>, type = <<"user">>},
                         reqid = <<"Xfh5mCQvjRgWDdlevrdyGt8M4lecXmN3gpGXrKKiUYqKdeD3">>,
                         organization_name = <<"cmwest">>,
-                        resource_state=#node_state{node_data = {[{<<"name">>,<<"db">>}]} }},
+                        resource_state=#node_state{node_data = {[{<<"name">>,<<"db">>},
+                                                                 {<<"password">>,<<"changeme">>}]} }},
```

Comment:
I would like to add a test to ensure that a redacted password doesn't get added to the data field if no password key is present. Not sure how to do it without too much code repetition.